<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacePage</class>
 <widget class="QWidget" name="InterfacePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>1121</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure User Interface&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Browsers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="uiBrowsersListFoldersFirstCheckBox">
        <property name="toolTip">
         <string>Select, if folders should be listed first in the various browsers</string>
        </property>
        <property name="text">
         <string>List folders first in Browsers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="uiBrowsersHideNonPublicCheckBox">
        <property name="toolTip">
         <string>Select to hide non public classes, methods and attributes in the browsers.</string>
        </property>
        <property name="text">
         <string>Hide non public members in Browsers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="uiBrowsersSortByOccurrenceCheckBox">
        <property name="toolTip">
         <string>Select to sort file contents by occurrence</string>
        </property>
        <property name="text">
         <string>Sort contents by occurrence</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="uiBrowsersShowHiddenCheckBox">
        <property name="toolTip">
         <string>Select to show hidden files in the various browsers</string>
        </property>
        <property name="text">
         <string>Show hidden files</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Filter out files:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileFiltersEdit">
          <property name="toolTip">
           <string>Enter wildcard file patterns separated by semicolon. Files matching these patterns will not be shown by the file browsers.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="uiCaptionShowsFilenameGroupBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Select, if the caption of the main window should show the filename of the current editor</string>
     </property>
     <property name="title">
      <string>Caption shows filename</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filename Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="filenameLengthSpinBox">
        <property name="toolTip">
         <string>Enter the number of characters to be shown in the main window title.</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>23</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Style:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="styleComboBox">
       <property name="toolTip">
        <string>Select the interface style</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Style Sheet:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="E5PathPicker" name="styleSheetPicker" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Enter the path of the style sheet file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dockarea Corner Usage</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Top Left Corner</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="tlTopButton">
           <property name="toolTip">
            <string>Select to assign the top left corner to the top dockarea</string>
           </property>
           <property name="text">
            <string>Top dockarea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tlLeftButton">
           <property name="toolTip">
            <string>Select to assign the top left corner to the left dockarea</string>
           </property>
           <property name="text">
            <string>Left dockarea</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Top Right Corner</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="trTopButton">
           <property name="toolTip">
            <string>Select to assign the top right corner to the top dockarea</string>
           </property>
           <property name="text">
            <string>Top dockarea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="trRightButton">
           <property name="toolTip">
            <string>Select to assign the top right corner to the right dockarea</string>
           </property>
           <property name="text">
            <string>Right dockarea</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Bottom Left Corner</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="blBottomButton">
           <property name="toolTip">
            <string>Select to assign the bottom left corner to the bottom dockarea</string>
           </property>
           <property name="text">
            <string>Bottom dockarea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="blLeftButton">
           <property name="toolTip">
            <string>Select to assign the bottom left corner to the left dockarea</string>
           </property>
           <property name="text">
            <string>Left dockarea</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Bottom Right Corner</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QRadioButton" name="brTopButton">
           <property name="toolTip">
            <string>Select to assign the bottom right corner to the bottom dockarea</string>
           </property>
           <property name="text">
            <string>Bottom dockarea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="brRightButton">
           <property name="toolTip">
            <string>Select to assign the bottom right corner to the right dockarea</string>
           </property>
           <property name="text">
            <string>Right dockarea</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Sidebars</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Delay:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="delaySpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="maximum">
         <number>2000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>396</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line9_2">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel1_2_2_2">
     <property name="text">
      <string>&lt;font color=&quot;#FF0000&quot;&gt;&lt;b&gt;Note:&lt;/b&gt; All settings below are activated at the next startup of the application.&lt;/font&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Language:</string>
       </property>
       <property name="buddy">
        <cstring>languageComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="languageComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the interface language.</string>
       </property>
       <property name="whatsThis">
        <string>The interface language can be selected from this list. If &quot;system&quot; is selected, the interface language is determined by the system. The selection of &quot;none&quot; means, that the default language will be used.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="layoutLabel">
       <property name="text">
        <string>Layout:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="layoutComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the layout type.</string>
       </property>
       <item>
        <property name="text">
         <string>Sidebars</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Toolboxes</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="title">
        <string>Shell</string>
       </property>
       <layout class="QVBoxLayout" name="_8">
        <item>
         <widget class="QRadioButton" name="separateShellButton">
          <property name="toolTip">
           <string>Select to get a separate shell window</string>
          </property>
          <property name="text">
           <string>separate window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="debugEmbeddedShellButton">
          <property name="toolTip">
           <string>Select to embed the shell in the Debug-Viewer</string>
          </property>
          <property name="text">
           <string>embed in Debug-Viewer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="title">
        <string>File-Browser</string>
       </property>
       <layout class="QVBoxLayout" name="_9">
        <item>
         <widget class="QRadioButton" name="separateFileBrowserButton">
          <property name="toolTip">
           <string>Select to get a separate file browser window</string>
          </property>
          <property name="text">
           <string>separate window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="debugEmbeddedFileBrowserButton">
          <property name="toolTip">
           <string>Select to embed the file browser in the Debug-Viewer</string>
          </property>
          <property name="text">
           <string>embed in Debug-Viewer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="projectEmbeddedFileBrowserButton">
          <property name="toolTip">
           <string>Select to embed the file browser in the Project-Viewer</string>
          </property>
          <property name="text">
           <string>embed in Project-Viewer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="tabsGroupBox">
     <property name="title">
      <string>Tabs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="tabsCloseButtonCheckBox">
        <property name="text">
         <string>Show only one close button instead of one for each tab</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>537</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="resetLayoutButton">
     <property name="text">
      <string>Reset layout to factory defaults</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiBrowsersListFoldersFirstCheckBox</tabstop>
  <tabstop>uiBrowsersHideNonPublicCheckBox</tabstop>
  <tabstop>uiBrowsersSortByOccurrenceCheckBox</tabstop>
  <tabstop>uiBrowsersShowHiddenCheckBox</tabstop>
  <tabstop>fileFiltersEdit</tabstop>
  <tabstop>uiCaptionShowsFilenameGroupBox</tabstop>
  <tabstop>filenameLengthSpinBox</tabstop>
  <tabstop>styleComboBox</tabstop>
  <tabstop>styleSheetPicker</tabstop>
  <tabstop>tlTopButton</tabstop>
  <tabstop>tlLeftButton</tabstop>
  <tabstop>trTopButton</tabstop>
  <tabstop>trRightButton</tabstop>
  <tabstop>blBottomButton</tabstop>
  <tabstop>blLeftButton</tabstop>
  <tabstop>brTopButton</tabstop>
  <tabstop>brRightButton</tabstop>
  <tabstop>delaySpinBox</tabstop>
  <tabstop>languageComboBox</tabstop>
  <tabstop>layoutComboBox</tabstop>
  <tabstop>separateShellButton</tabstop>
  <tabstop>debugEmbeddedShellButton</tabstop>
  <tabstop>separateFileBrowserButton</tabstop>
  <tabstop>debugEmbeddedFileBrowserButton</tabstop>
  <tabstop>projectEmbeddedFileBrowserButton</tabstop>
  <tabstop>tabsCloseButtonCheckBox</tabstop>
  <tabstop>resetLayoutButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
